scss/imports/_header.scss

```diff
@@ -0,0 +1,23 @@
+// dat-header
+
+.dat-header {
+  height: 3rem;
+  padding: .5rem .75rem;
+  background-color: $color-neutral;
+  color: $color-white;
+  -webkit-app-region: drag;
+}
+
+.menu-trigger {
+  height: 2rem;
+  padding-left: .625rem;
+  width: 2.5rem;
+  .icon-menu {
+    fill: $color-neutral--20;
+  }
+  &:hover, &:focus {
+    .icon-menu {
```

Comment:
I do find these statements a bit hard to read, I think this translates to:
```css
.menu-trigger:hover .icon-menu,
.menu-trigger:focus .icon-menu {}
```
right? Just wanna make sure I understand what's going on alright haha